docs/deploy_fix.md

```diff
@@ -0,0 +1,31 @@
+# Potential error messages when Deploying the template locally
+
+This section covers common error messages you might encounter when deploying templates locally, along with solutions to resolve them.
+
+## Error message: Quota and resources available: InsufficientQuota:
+ _This operation require 30 new capacities in quota Tokens Per Minute (thousands) - GPT-35-Turbo, which is bigger than the current available capacity 0. The current quota usage is 30 and the quota limit is 30 for quota Tokens Per Minute (thousands) - GPT-35-Turbo._
+
+Possible solutions: 
+
+1.	Review and adjust the quota: Check if you can increase the quota for Tokens Per Minute (thousands) for your gpt-model in your Azure settings. This might involve requesting a quota increase through the Azure portal.
+2.	Optimize token usage: Examine your implementation to see if you can reduce the number of tokens you're using. 
```

Comment:
This would not work. Our bicep requests 30, so it doesnt matter what the usage is like.
However, they can set this environment variable if they have less than 30 TPM available:

```
azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT_CAPACITY 20
```

Reply:
Okidoki, would you recommed to include it as an option here on the file?